README.md

```diff
@@ -19,12 +18,13 @@ Foundation to Professor Robert Ewers:
 * [NOMIS Award details](https://nomisfoundation.ch/people/robert-ewers/)
 * [NOMIS project summary](https://nomisfoundation.ch/research-projects/a-virtual-rainforest-for-understanding-the-stability-resilience-and-sustainability-of-complex-ecosystems/)
 
-[<img alt="NOMIS logo" src="https://nomisfoundation.ch/wp-content/uploads/2020/05/nomis-foundation-logo-372-by-151.png" width=250>](https://nomisfoundation.ch)
+<!-- markdownlint-disable-next-line  MD033 MD013-->
+[<img alt="NOMIS logo" src="docs/source/_static/images/logo-nomis-822-by-321.png" width=250>](https://nomisfoundation.ch)
```

Comment:
Just a comment for me to check I can make comments. Looks okay to me.

Reply:
It's ok, I cna see you :)